test/aqua.jl

```diff
@@ -1,27 +1,35 @@
 using Test
-import ClimaCore
+using ClimaCore
 using Aqua
 
-@testset "Aqua tests - unbound args" begin
+@testset "Aqua tests (performance)" begin
     # This tests that we don't accidentally run into
     # https://github.com/JuliaLang/julia/issues/29393
-    Aqua.test_unbound_args(ClimaCore)
+    # Aqua.test_unbound_args(ClimaCore)
```

Comment:
Should this line be kept as a comment or just removed?

Reply:
Yes, we can just remove it